lib/car.rb

```diff
@@ -0,0 +1,26 @@
+require 'pry'
+class Car
```

Comment:
You are missing `color` attribute and `paint!` method defined in Iteration 1 spec.